x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/search_experience.tsx

```diff
@@ -85,12 +97,25 @@ export const SearchExperience: React.FC = () => {
               sortOptions={sortingOptions}
               view={SortingView}
             />
+            <EuiSpacer />
+            <CustomizationCallout onClick={() => setShowCustomizationModal(true)} />
```

Comment:
optional, I'll probably request this later in your next PR / once you add the filters in - we should DRY this function out to a `const openCustomizationModal = () => setShowCustomizationModal(true)` var since there's at least one other instance of this same fn in the source code. I'd put it between line 49 and 50

Reply:
I actually think that change makes the code less readable. I think it's easier to jump through the code and see the various instance of `setShowCustomizationModal` being called either with true or false in a consistent way, rather than having to jump up the file to that function definition.

If you think that your way is objectively better for some reason or another, let me know and I'll make that change.

Reply:
mostly a microperf thing, if you strongly think it's more readable I don't feel strongly enough to oppose it, but personally DRYing out two `openCustomizationModal` and `closeCustomizationModal` fns feels a bit more readable to me 🤷

Reply:
c30007527adb30a6220ab23186b1c9ac8d0f27a3

Reply:
f637941fc61a7f596deab2e9c9a7800d0c530493